### ISSUE TITLE
Develop a Power BI usage document for users.

### ISSUE THREAD

**Issue by leelakrishna-t** [ServAcuity/ServAcuity#101]

**Comment by leelakrishna-t** [comment]
This will cover basic functionalities like how to use filters, navigate the report, export reports to PowerPoint, and generate data insights within the Power BI service etc.

**Comment by naveen-biz** [comment]
I've started preparing the document for users on how to effectively use Power BI, focusing on the user perspective. The document covers key aspects such as applying filters, understanding visuals, navigating between the homepage and report pages, accessing Power BI, refreshing data, interacting with visuals and cross-filtering, exporting data, understanding KPIs, and sharing reports. I have ensured to include all features that would be useful for the users.

**Comment by naveen-biz** [comment]
I've sent the document to Leela for review, and he suggested adding snapshots, which I have incorporated. I also included the ServAcuity logo and footer line colors for a more polished design. After the second iteration, Leela reviewed it again, and the document has now been moved to live. The file has been stored in SharePoint for access.[How to Power BI.docx](https://github.com/user-attachments/files/17473397/How.to.Power.BI.docx)
